spec/functional/application_spec.rb

```diff
@@ -16,14 +16,14 @@
   end
 
   def send_example
-    send_message(fixture('example01')).should be_truthy
+    send_message(fixture('example01'))
```

Comment:
Because we're `raise`ing below, we will emit `nil` if all is well; but we don't need to test that.